app/event_source/subscribers/iap/application_determined_subscriber.rb

```diff
@@ -14,7 +14,8 @@ class ApplicationDeterminedSubscriber
       # create_tax_household_group(subscriber_logger, payload) if !Rails.env.test? && EnrollRegistry.feature_enabled?(:temporary_configuration_enable_multi_tax_household_feature)
       # generate_enrollments(subscriber_logger, payload) if !Rails.env.test? && EnrollRegistry.feature_enabled?(:temporary_configuration_enable_multi_tax_household_feature)
 
-      update_application_evidence_histories(payload)
+      # This method was moved to components/financial_assistance/app/domain/financial_assistance/operations/applications/verifications/request_evidence_determination.rb
```

Comment:
@TristanB17 - any blockers to move logic from `request_evidence_determination.rb` to this subscriber instead of above? (this way, we can continue using event source!)

Reply:
@nisanthyaganti9 yeah, originally I did move the logic around to accommodate leaving this method in event_source, but there were too many edge cases, namely the malformed CV3 application needs to log verification histories to all evidences, which would result in us not being able to log the application_determination history in any of the evidences.